src/PrestaShopBundle/Form/Admin/Improve/International/Tax/TaxType.php

```diff
@@ -0,0 +1,130 @@
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Form\Admin\Improve\International\Tax;
+
+use PrestaShopBundle\Form\Admin\Type\SwitchType;
+use PrestaShopBundle\Form\Admin\Type\TranslatableType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Constraints\Regex;
+use Symfony\Component\Validator\Constraints\Type;
+
+/**
+ * Form type for tax add/edit
+ */
+class TaxType extends AbstractType
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('name', TranslatableType::class, [
+                'options' => [
+                    'constraints' => [
+                        new Regex([
+                            'pattern' => '/^[^<>={}]*$/u',
+                            'message' => $this->translator->trans(
+                                '%s is invalid.',
+                                [
+                                    sprintf('"%s"', $this->translator->trans('Name', [], 'Admin.Global')),
+                                ],
+                                'Admin.Notifications.Error'
+                            ),
+                        ]),
+                        new NotBlank([
+                            'message' => $this->translator->trans(
+                                'The %s field is required.',
+                                [
+                                    sprintf('"%s"', $this->translator->trans('Name', [], 'Admin.Global')),
+                                ],
+                                'Admin.Notifications.Error'
+                            ),
+                        ]),
+                        new Length([
+                            'max' => 32,
+                            'maxMessage' => $this->translator->trans(
+                                'This field cannot be longer than %limit% characters',
+                                ['%limit%' => 32],
+                                'Admin.Notifications.Error'
+                            ),
+                        ]),
+                    ],
+                ],
+            ])
+            ->add('rate', TextType::class, [
+                'constraints' => [
+                    new Length([
+                        'max' => 6,
+                        'maxMessage' => $this->translator->trans(
+                            'This field cannot be longer than %limit% characters',
```

Comment:
Same question here. ;-)

Reply:
Same answer here, let's keep your suggestion: _This field cannot be longer than %limit% characters_ in _Admin.Notifications.Error_.